exp/api/v1beta1/azuremachinepool_webhook_test.go

```diff
@@ -263,10 +263,12 @@ func TestAzureMachinePool_Default(t *testing.T) {
 func createMachinePoolWithtMarketPlaceImage(publisher, offer, sku, version string, terminateNotificationTimeout *int) *AzureMachinePool {
```

Comment:
there is a typo in the function name if you want to fix it while you're here :)

```suggestion
func createMachinePoolWithMarketPlaceImage(publisher, offer, sku, version string, terminateNotificationTimeout *int) *AzureMachinePool {
```